### PR TITLE
Move setup dependencies to pyproject.yaml

### PR DESCRIPTION
**Pull request by raddessi**
From https://github.com/smarie/python-pytest-cases/issues/65

Same issue here, my editor blacked setup.py. Just let me know if you want me to revert.

### CODE REVIEW

**Comment by smarie** [comment]
and same feedback :) yes please revert setup.py entirely, that will be better for the PR.

**Comment by raddessi** [comment]
No problem :) Done, thanks

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[build-system]
+requires = [
+    "setuptools",
+    "setuptools_scm",
+    "six",
+]
+build-backend = "setuptools.build_meta"
```
Comment (smarie) [inline pyproject.toml]: Not useful as per https://github.com/pypa/pip/issues/6453

```suggestion
```

**Comment by smarie** [comment]
thanks!

**Pull request merged**